Servers/Administration/OJS.Servers.Administration/Controllers/UserRolesController.cs

```diff
@@ -1,17 +1,24 @@
 namespace OJS.Servers.Administration.Controllers;
 
+using AutoCrudAdmin.Enumerations;
 using AutoCrudAdmin.Extensions;
 using AutoCrudAdmin.ViewModels;
 using Microsoft.AspNetCore.Authorization;
 using OJS.Data.Models.Users;
 using System;
 using System.Collections.Generic;
 using System.Linq.Expressions;
-using static OJS.Common.GlobalConstants.Roles;
+using static Common.GlobalConstants.Roles;
+using OJS.Services.Administration.Data;
+using System.Linq;
 
 [Authorize(Roles = Administrator)]
 public class UserRolesController : BaseAutoCrudAdminController<UserInRole>
 {
+    private readonly IUsersDataService usersDataService;
+
+    public UserRolesController(IUsersDataService usersDataService) => this.usersDataService = usersDataService;
```

Comment:
You can put `=> this.usersDataService = usersDataService;` on a new row for better code clearness.

Reply:
Done.